docs/general/installation/container.md

```diff
@@ -19,6 +19,8 @@ hotio image: `ghcr.io/hotio/jellyfin`.
 Jellyfin distributes official container images on [Docker Hub](https://hub.docker.com/r/jellyfin/jellyfin/) and the [GitHub Container Registry](https://ghcr.io/jellyfin/jellyfin) for multiple architectures.
 These images are based on Debian and [built directly from the Jellyfin source code](https://github.com/jellyfin/jellyfin-packaging/blob/master/docker/Dockerfile).
 
+Several tags are available tracking different builds and version levels as described in this [forum post](https://forum.jellyfin.org/t-new-jellyfin-server-web-release-10-9-6?pid=25895#pid25895). Make sure you understand the implications of the tag you choose if using an automated update mechanism like watchtower, including that a [backup](https://jellyfin.org/docs/general/administration/backup-and-restore) is required to restore a previous version.
```

Comment:
The forum is not a documentation source. Specify the tags directly here in the docs instead.